src/Files.App/Dialogs/GitHubLoginDialog.xaml

```diff
@@ -0,0 +1,76 @@
+﻿<!--  Copyright (c) 2023 Files Community. Licensed under the MIT License. See the LICENSE.  -->
+<ContentDialog
+	x:Class="Files.App.Dialogs.GitHubLoginDialog"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:converters="using:CommunityToolkit.WinUI.UI.Converters"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:helpers="using:Files.App.Helpers"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	Title="{helpers:ResourceString Name=ConnectGitHub}"
+	CloseButtonStyle="{StaticResource AccentButtonStyle}"
+	CloseButtonText="{helpers:ResourceString Name=Close}"
+	CornerRadius="{StaticResource OverlayCornerRadius}"
+	PrimaryButtonClick="ContentDialog_PrimaryButtonClick"
+	PrimaryButtonText="{helpers:ResourceString Name=OK}"
+	RequestedTheme="{x:Bind helpers:ThemeHelper.RootTheme}"
+	Style="{StaticResource DefaultContentDialogStyle}"
+	mc:Ignorable="d">
+
+	<ContentDialog.PrimaryButtonStyle>
+		<Style BasedOn="{StaticResource DefaultButtonStyle}" TargetType="Button">
+			<Setter Property="ContentTemplate">
+				<Setter.Value>
+					<DataTemplate>
+						<StackPanel
+							HorizontalAlignment="Center"
+							Orientation="Horizontal"
+							Spacing="8">
+							<FontIcon FontSize="14" Glyph="&#xE8C8;" />
+							<TextBlock Text="{helpers:ResourceString Name=CopyCode}" />
+						</StackPanel>
+					</DataTemplate>
+				</Setter.Value>
+			</Setter>
+		</Style>
+	</ContentDialog.PrimaryButtonStyle>
+
+	<ContentDialog.Resources>
+		<ResourceDictionary>
+			<converters:BoolNegationConverter x:Key="BoolNegationConverter" />
+		</ResourceDictionary>
+	</ContentDialog.Resources>
+
+	<Grid RowSpacing="20">
+		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto" />
+			<RowDefinition />
+		</Grid.RowDefinitions>
+
+		<!--  Subtitle  -->
+		<TextBlock
```

Comment:
Isn't there a property in ContentDialog for this?

Reply:
It doesn't seem so, we're doing the same thing in `DynamicDialog`